scripts/report_errors.sh

```diff
@@ -44,6 +44,28 @@ function post_error_to_github
       --header 'content-type: application/json' \
       -u stackenbotten:$GITHUB_ACCESS_TOKEN \
       --data "{\"body\": \"There was an error when running \`$CIRCLE_JOB\` for commit \`$CIRCLE_SHA1\`:\n\`\`\`\n$FORMATTED_ERROR_MSG\n\`\`\`\nPlease check that your changes are working as intended.\"}"
+
+    post_review_comment_to_github
+}
+
+function post_review_comment_to_github
+{
+    GITHUB_API_URL="https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/pulls/$CIRCLE_PR_NUMBER/comments"
+
+    sed -i 1d $ERROR_LOG
+
+    while read line
+    do
+        ERROR_PATH=$(echo $line | grep -oE ".*\.cpp")
```

Comment:
I think the regexes here can still be improved, but I'm merging for now. Thanks a lot for your help!